packages/amplify-cli/templates/new-plugin-frontend/amplify-plugin.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
"new-plugin-frontend" dir name doesn't make sense - and is not maintainable.

Reply:
not getting it. how would you like to name it?